source/_posts/2017-01-21-home-assistant-governance.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Home Assistant Governance"
+description: "Protecting Home Assistant for all, now and into the future"
+date: 2017-01-21 15:05:00 -0800
+date_formatted: "January 21, 2017"
+author: Paulus Schoutsen
+author_twitter: balloob
+comments: true
+categories: Organisation
+---
+
+The Home Assistant project and community has seen enormous growth in the last three years. Hundreds, if not thousands, of volunteers have and are working tirelessly every day on giving you all the awesomeness that you see today. That's not just the code but also all the related projects, documentation, examples, release blog posts and help in our forums and chat. And this is something that we want to keep safe and functional, even as we grow.
+
+Starting today we are announcing a few initiatives to help protect our users, contributors and community members.
+
+## Code of Conduct
+
+More and more people are getting to know and love Home Assistant. Our community keeps growing and growing. Our community consists of people from all over the world with different backgrounds and we want Home Assistant to be a place where everyone can feel at home. To help with this we're introducing a Code of Conduct. The Code of Conduct describes what type of behavior is unacceptable, how we will enforce it and where to report incidents.
+
+Moving forward this Code of Conduct will be active for all projects within the Home Assistant organisation.
+
+[Read the Code of Conduct][coc]
+
+[coc]: http://contributor-covenant.org/version/1/4/
+
+## Contributor License Agreement
+
+Moving forward we will also require that all contributors to any project have electronically signed a contributor license agreement (CLA). We have adopted the CLA that GitHub uses for their projects. Currently we have no plans to retroactively require a CLA for previous contributions, only those going forward.
+
+Until a signed CLA is on file your pull request will not be reviewed or accepted. Simply open a new pull request and if a CLA does not exist for you our helpful bot will walk you through the steps to complete the process.
```

Comment:
`Simply open a new pull request and if a CLA does not exist for you our helpful bot will walk you through the steps to complete the process.`

Fragment.

`When you open a new pull request, if you do not have a CLA on file, our helpful bot will walk you through the steps to complete the process.`